pages/main.js

```diff
@@ -0,0 +1,45 @@
+document.querySelector(".login_form").addEventListener("submit", generateTable);
+
+    function generateTable(event) {
+     event.preventDefault();
+
+     console.log(12123);
+
+     const firstName = document.getElementById("first-name").value;
+     const lastName = document.getElementById("last-name").value;
+     const birthYear = document.getElementById("birth-year").value;
+     const sex = document.querySelectorAll('input[name="sex"]:checked').value;
+    
+     const city = document.getElementById("city").value;
+     const address = document.getElementById("address").value;
+
+     const languages = [];
+
+     console.log([firstName, lastName, city, sex, birthYear, address, languages]);
+
+     const languageCheckboxes = Array.from(document.getElementsByName("language"));
+     console.log(languageCheckboxes);
+
+      for (var i = 0; i < languageCheckboxes.length; i++) {
+        if (languageCheckboxes[i].checked) {
+          languages.push(languageCheckboxes[i].value);
+        }
+      }
+    
+      const table = document.createElement("table");
+
+      table.setAttribute("id", "user-table");
+      
+      table.innerHTML = "<tr><td>Ім'я:</td><td>" + firstName + "</td></tr>" +
```

Comment:
Use template string, it is easier and prettier
You won't need these strange pluses

```
table.innerHTML = (
  `
  <tr>
    <td>
      Ім'я:
    </td>
    <td>
      ${firstName}
    </td>
  </tr>
  <tr>
    // the same for others
  </tr>
  `
  );
```